tests/bin/import-metadata

```diff
@@ -41,5 +41,5 @@ for F in $DNF_DOWNLOAD/*.rpm; do
 done
 
 # cleanup temporary directories and files
-rm -rf $DNF_ROOT $DNF_DOWNLOAD $IMPORT $SCHEMA
-[ "$REMOVE_IMPORT_REPO" == 1 ] && rm -rf $IMPORT_REPO
+sudo rm -rf $DNF_ROOT $DNF_DOWNLOAD $IMPORT $SCHEMA
```

Comment:
This makes me *really* nervous. I'd like to either see some sanity checks on the variables or see them used in combination with a fixed base directory so that we don't somehow accidentally end up running sudo rm -rf / or something similar.